### PR TITLE
https://issues.redhat.com/browse/ACM-4685 Subm prereq

### PR DESCRIPTION
**Pull request by oafischer**
2.8, 2.7, 2.6, 2.5

### CODE REVIEW

**Review by cadawson — APPROVED** [review]
Looks good. Thanks!

File: add-ons/submariner/subm_configure.adoc [hunk 1]
```diff
@@ -10,9 +10,10 @@ Ensure that you have the following prerequisites before using Submariner:
 
 * A credential to access the hub cluster with `cluster-admin` permissions.
 * IP connectivity must be configured between the Gateway nodes. When connecting two clusters, at least one of the clusters must be accessible to the Gateway node using its public or private IP address designated to the Gateway node. See https://submariner.io/operations/nat-traversal[Submariner NAT Traversal] for more information.
+* If the Gateway nodes are directly reachable over their private IPs without any NAT in between, make sure that the firewall configuration allows the ESP protocol on the Gateway nodes.
```
Comment (sridhargaddam) [inline add-ons/submariner/subm_configure.adoc]: @oafischer Can you please move this to line 17. Basically, the main requirement is to convey that 4500/UDP on the gateway nodes should be opened. After talking about that, lets add this point below that line.

**Review by sridhargaddam — COMMENTED** [review]
A small comment.

File: add-ons/submariner/subm_configure.adoc (hunk 1 above)
Comment (oafischer) [reply add-ons/submariner/subm_configure.adoc]: Updated, thanks for the review!

**Review by sridhargaddam — APPROVED** [review]
Thanks @oafischer

**Pull request merged**